continuous_integration/recipe/conda_build_config.yaml

```
@@ -1,4 +1,4 @@
 rust_compiler_version:
-    - 1.69
+    - 1.73
 maturin:
     - 1.1
```

Should change this to match whatever pinned maturin version we converge on; assuming for now that's 1.3.1:


```suggestion
    - 1.3.1
```